src/components/HeatmapCell.js

```diff
@@ -47,14 +47,12 @@ const HeatmapCell = ({value, state, info}) => {
   }
   
   if (state === heatmapCellStates.isDimmed) return (
-    <div className='group container flex relative w-[100%] justify-center 
```

Comment:
Since all 3 conditions return this div with the same group container flex styling, why not have your HeatmapCell do something like:

return (
    <div className="group container flex relative w-[100%] justify-center>
         {getCellState()}
    </div>
)